File: NitKotin.Android/app/src/main/java/com/nitkotin/android/data/model/SmokingConfig.kt
package com.nitkotin.android.data.model

import java.time.Instant

data class SmokingConfig(
    val languagePreference: AppLanguage = AppLanguage.ENGLISH,
    val hasStartedTracking: Boolean = false,
    val quitDateTime: Instant = Instant.now(),
    val packsPerDay: Double = 2.0,
    val packPriceUah: Double = 160.0,
)

File: NitKotin/Models/SmokingConfig.cs
namespace NitKotin.Models;

public sealed class SmokingConfig
{
    public string LanguagePreference { get; set; } = "en";

    public bool HasStartedTracking { get; set; }

    public DateTime QuitDateTime { get; set; } = DateTime.Now;

    public decimal PacksPerDay { get; set; } = 2m;

    public decimal PackPriceUah { get; set; } = 160m;

    public int OverlayLocationX { get; set; } = -1;

    public int OverlayLocationY { get; set; } = -1;
}